util/utils.hpp

```diff
@@ -25,37 +25,43 @@ namespace ts { // ts = Tensor Sketch
  * @return the extracted kmers, as integers converted from base #alphabet_size
  */
 template <class chr, class kmer>
-std::vector<kmer> seq2kmer(const std::vector<chr> &seq, uint8_t kmer_size, uint8_t alphabet_size) {
+std::vector<kmer> seq2kmer(const std::vector<chr> &seq, size_t kmer_size, size_t alphabet_size) {
```

Comment:
Do you think we'll ever have a kmer or alphabet size that are above 255?

Reply:
alphabet size, yes it can grow to larger sets. kmer size probably not. My main argument is that  unit8_t doesn't offer any memory/time benefits, but in the worst case it can overflow and become a hard to detect bug

Reply:
IMHO those benefits are not relevant to the change I've made here, these are just argument types. We can store the sequences in `vector<uint8_t>` and  `alphabet_size` in `uint64_t` at the same time. Having said that, I don't have a strong objection to uint8_t here, if you insist.